### PR TITLE
OCLOMRS-316: Make the add existing concept table to be like the concepts display table

### PR DESCRIPTION
**Pull request by dessHub**
# JIRA TICKET NAME:
[OCLOMRS-316: Make the add existing concept table to be like the concepts display table](https://issues.openmrs.org/browse/OCLOMRS-316)

# Summary:
Update addBulkConcepts tables to look like the concepts display table.

### CODE REVIEW

File: src/components/bulkConcepts/bulkConceptList.jsx [hunk 1]
```diff
@@ -1,29 +1,66 @@
...
-            <BulkConceptTable concept={concept} key={concept.id} handleSelect={handleSelect} />
-          ))}
-        </tbody>
-      </table>
+      <ReactTable
+        data={cielConcepts}
+        loading={fetching}
+        defaultPageSize={cielConcepts.length <= conceptLimit ? cielConcepts.length : conceptLimit}
+        filterable
+        noDataText="No ceoncept!"
```
Comment (abulojoshua1) [inline src/components/bulkConcepts/bulkConceptList.jsx]: @dessHub, I think you meant `No concept!` here. Please take a look at the spelling.

File: src/components/bulkConcepts/component/ConceptTable.jsx [hunk 2]
```diff
@@ -1,52 +1,76 @@
...
-              />
-            ))}
-          </tbody>
-        </table>
+        <ReactTable
+          data={concepts}
+          loading={loading}
+          defaultPageSize={concepts.length <= conceptLimit ? concepts.length : conceptLimit}
+          filterable
+          noDataText="No ceoncept!"
```
Comment (abulojoshua1) [inline src/components/bulkConcepts/component/ConceptTable.jsx]: @dessHub, I think you meant No concept! here. Please take a look at the spelling.

File: src/components/bulkConcepts/bulkConceptList.jsx (hunk 1 above)
Comment (dessHub) [reply src/components/bulkConcepts/bulkConceptList.jsx]: Ooh yeah, on it

File: src/tests/Dictionary/DictionaryContainer.test.jsx [hunk 3]
```diff
@@ -88,7 +88,7 @@ describe('DictionaryOverview', () => {
   });
 
   it('should handle HEAD version release', () => {
-    const props = {
+    const  props = {
```
Comment (Duncanian) [inline src/tests/Dictionary/DictionaryContainer.test.jsx]: @dessHub , why the extra space before `props`?

File: src/components/bulkConcepts/bulkConceptList.jsx (hunk 1 above)
Comment (dessHub) [reply src/components/bulkConcepts/bulkConceptList.jsx]: Fixed it.

File: src/components/bulkConcepts/component/ConceptTable.jsx (hunk 2 above)
Comment (dessHub) [reply src/components/bulkConcepts/component/ConceptTable.jsx]: Fixed it.

File: src/tests/Dictionary/DictionaryContainer.test.jsx (hunk 3 above)
Comment (dessHub) [reply src/tests/Dictionary/DictionaryContainer.test.jsx]: Fixed it.

File: src/components/bulkConcepts/addBulkConcepts.jsx [hunk 4]
```diff
@@ -62,10 +65,16 @@ export class AddBulkConcepts extends Component {
     const { id } = e.target;
     this.setState({ currentPage: Number(id) });
   };
+
+  filterCaseInsensitive = (filter, rows) => {
```
Comment (dondrzzy) [inline src/components/bulkConcepts/addBulkConcepts.jsx]: What does `filterCaseInsensitive` mean @dessHub?

Comment (dessHub) [reply src/components/bulkConcepts/addBulkConcepts.jsx]: @dondrzzy It means it does filter based on the given input regardless of the input case. For instance, `Random` will match `random` and `Random`.

**Comment by dkayiwa** [comment]
@dondrzzy do you have any further comments on this?

**Pull request merged**